README.md

```diff
@@ -191,3 +200,12 @@ This code is released under the MIT License. See [LICENSE.txt](/LICENSE.txt).
 - Introduce code verification using something like GPG signatures or published checksums
 - Explicitly test for exotic repo and org names
 - Apply stricter parsing for repo-filter command-line arg
+
+## Issue #73
```

Comment:
I assume these are just your own notes?

Reply:
Ah, saw your message on Slack. Ignore this.

Reply:
Yes, the implementation plan for all to enjoy. Certainly will be removed at the end!